pkg/integration/integration.go

```diff
@@ -108,23 +104,13 @@ func (*Integration) Deactivate(name string, namespace string) error {
 
 	activeFilters := viper.GetStringSlice("active_filters")
 
-	// Update filters
-	// This might be a bad idea, but we cannot reference analyzer here
-	foundFilter := false
-	for i, v := range activeFilters {
-
-		for _, intanal := range integrations[name].GetAnalyzerName() {
-			if v == intanal {
-				foundFilter = true
-				activeFilters = append(activeFilters[:i], activeFilters[i+1:]...)
-				break
+	// Update filters and remove the specific filters for the integration
+	for _, filter := range integrations[name].GetAnalyzerName() {
```

Comment:
What happens if I attempt to update or remove an unactivated filter?

Reply:
This is true.. it's only parsing the active filters, will need to check the deactivated ones also, good spot

Reply:
I think this would be okay upon second thought. 
The unused filter is only listed from the integration filters, which would go once the integration is deactivated